sdk/storage/Azure.Storage.Blobs/api/Azure.Storage.Blobs.netstandard2.0.cs

```diff
@@ -32,6 +32,8 @@ public partial class BlobClient : Azure.Storage.Blobs.Specialized.BlobBaseClient
         public virtual System.Threading.Tasks.Task<Azure.Response<Azure.Storage.Blobs.Models.BlobContentInfo>> UploadAsync(string path, Azure.Storage.Blobs.Models.BlobUploadOptions options, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public virtual System.Threading.Tasks.Task<Azure.Response<Azure.Storage.Blobs.Models.BlobContentInfo>> UploadAsync(string path, bool overwrite = false, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public virtual System.Threading.Tasks.Task<Azure.Response<Azure.Storage.Blobs.Models.BlobContentInfo>> UploadAsync(string path, System.Threading.CancellationToken cancellationToken) { throw null; }
+        public new Azure.Storage.Blobs.BlobClient WithSnapshot(string snapshot) { throw null; }
```

Comment:
BlobClient.WithSnapshot() and .WithVersion() don't currently work without this.

Reply:
Do you mean  BlobClient.WithSnapshot() and .WithVersion() don't work after the fix?
But I see you have test case below for BlobClient on the 2 APIs.

Reply:
They didn't work before the fix.
`BlobClient snapshotBlobClient = blobClient.WithSnapshot("...");`
`BlobClient versionBlobClient = blobClient.WithVersion("...");`
`Cannot implicitly convert type Azure.Storage.Blobs.Specialized.BaseBlobClient to Azure.Storage.Blobs.BlobClient...`